modules/base/src/components/SPN.vue

```diff
@@ -30,6 +30,14 @@
 
       <!-- FAQ -->
       <div class="doubling two column row spn-faq">
+        <div class="column">
+          <h4>Is the SPN out yet?</h4>
+          <p>The SPN is currently in a closed pre-alpha with our earliest Kickstarter backers (MARVIN & VISIONARY). A status page to inform you of the networks status, its size and number of users is planned. We will link to it from here as soon as it is out.</p>
```

Comment:
The HTML code for & is `&amp;`

```suggestion
          <p>The SPN is currently in a closed pre-alpha with our earliest Kickstarter backers (MARVIN &amp; VISIONARY). A status page to inform you of the networks status, its size and number of users is planned. We will link to it from here as soon as it is out.</p>
```